### PR TITLE
flow added

### PR DESCRIPTION
**Pull request by yasintazeoglu**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,8 +1,10 @@
 {
     "extends": [
 			"airbnb-base",
-			"plugin:react/recommended"
-		],
+      "plugin:react/recommended"
```
Comment (SelmanKahya) [inline .eslintrc.json]: indentation issue here?

File: razzle.config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (SelmanKahya) [inline razzle.config.js]: do we need this?

File: src/client.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
+import App from '@/containers/app';
 
 // eslint-disable-next-line no-underscore-dangle
 const store = setupStore(window.__PRELOADED_STATE__);
-
+const getElement = document.getElementById('root');
 hydrate(
-  <BrowserRouter>
-    <Provider store={store}>
+  <Provider store={store}>
```
Comment (SelmanKahya) [inline src/client.js]: why did you swap these lines?

File: src/client.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ if (module.hot) {
       <Provider store={store}>
         <App />
       </Provider>,
-      document.getElementById('root'),
+      getElement,
```
Comment (SelmanKahya) [inline src/client.js]: were you getting flow errors here?

File: src/components/pages/index.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import Home from './home';
+import Login from './login';
+import Logout from './logout';
+import MemberProfile from './memberProfile';
+
+export {
+  Home, Login, Logout, MemberProfile,
```
Comment (SelmanKahya) [inline src/components/pages/index.js]: lets remove index.js, keep it the old way

File: src/index.js [hunk 6]
```diff
@@ -11,18 +11,17 @@ server.listen(process.env.PORT || 3000, (error) => {
     console.log(error);
   }
 
+  if (module.hot) {
```
Comment (SelmanKahya) [inline src/index.js]: why did you put this here?

**Comment by SelmanKahya** [comment]
lets rm yarn.lock, and add it to gitignore. i think you already did it in dev branch, you can pull latest.

File: src/client.js (hunk 3 above)
Comment (yasintazeoglu) [reply src/client.js]: https://redux.js.org/advanced/usagewithreactrouter

**Comment by SelmanKahya** [comment]
resolved conflicts, waiting for travis build.

**Comment by SelmanKahya** [comment]
naisuuu

**Pull request merged**